pkg/controllers/node/controller.go

```diff
@@ -42,7 +42,7 @@ const controllerName = "node"
 func NewController(kubeClient client.Client) *Controller {
 	return &Controller{
 		kubeClient: kubeClient,
-		liveness:   &Liveness{kubeClient: kubeClient},
+		startup:    &Startup{kubeClient: kubeClient},
```

Comment:
Thoughts on "initialization". I know it deviates a bit from `startupProbe`, but I think it's a more accurate term, and we're already not perfectly following the convention.

Reply:
I don't mind. I'll change this to `initialization`